src/vsr/replica.zig

```diff
@@ -6492,36 +6532,34 @@ pub fn ReplicaType(
 ///
```

Comment:
Unrelated to the PR, but I continue to be mystified by the canonical terminology as it is explained here. 

My take on the explanation: 

Canonical messages: messages from DVC quorum with the highest log view. By the quorum intersection property, each op that could have been committed is included in at least one canonical message

Canonical headers: subset of headers from canonical messages, which could have been committed. This might be a strict subset: even if a header is present in some canonical message, we might conclude that it is not committed if it is missing from enough other canonical messages.

Reply:
>I consider "canonical headers" to be the union of all headers from all canonical messages.

But that's not how headers_canonical actually works, right? It strips uncommitted

Reply:
Good point. I guess I need to update the docs _and_ my mental model. :stuck_out_tongue:

Reply:
I ended up leaving my mental model intact, but renaming `headers_canonical()` to `quorum_headers()`.
This is more consistent overall — there is a more intuitive correspondence between the canonical headers/messages/view.